docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,25 +9,44 @@ ms.topic: conceptual
 
 # Select Assemblies Referenced By Projects
 
-Explicit assembly references allows a subset of assemblies to be used for IntelliSense and compiling, while all assemblies are available at run-time. `PackageReference` and `packages.config` work differently, and as a result package authors need to take care to create the package to be compatible with both project types.
+Assemblies are used in two different ways during a build. The first is for compile, which allows the package consumer's code to compile against APIs in the assembly, and for Intellisense to give suggestions. The second is runtime, where the assembly is copied to the `bin` directory and is used at runtime.
 
-> [!Note]
-> Explicit assembly references are related to .NET assemblies. It is not a method to distribute native assemblies that are P/Invoked by a managed assembly.
+## Recommendation: Package dependencies and include/exclude assets
 
-## `PackageReference` support
+Our recommendation is to have one package per assembly, in which case you can add `compile` to the private assets. Note that the default private assets when none is supplied is `build;analyzers`. Therefore, you should use `PrivateAssets="compile;build;analyzers"` in your `PackageReference` or `ProjectReference`. This provides more flexibility and minimizes risk that two different packages will contain different versions of the same assembly, preventing NuGet from being able to manage versions.
```

Comment:
> This provides more flexibility and minimizes risk that two different packages will contain different versions of the same assembly, preventing NuGet from being able to manage versions

Not sure this flows the same way you wanted it to.

Is this good or bad? I'd argue it's not great if you prevent NuGet from managing the versions.